cache/refs.go

```diff
@@ -32,33 +30,19 @@ import (
 // Ref is a reference to cacheable objects.
 type Ref interface {
 	Mountable
-	ID() string
+	Metadata
 	Release(context.Context) error
-	Size(ctx context.Context) (int64, error)
-	Metadata() *metadata.StorageItem
-	IdentityMapping() *idtools.IdentityMapping
```

Comment:
Why is `IdentityMapping` removed. It is not fully connected (used atm only in moby) but seems like required data

Reply:
I guess this can be accessed after `Mount` is called from `snapshot.Mountable`. Not sure if this is enough for all cases or something might need this info without actually doing a mount.

Reply:
Yeah I didn't see that it was ever used so I got rid of it. If moby needs it though then I'll just re-add it, not a big deal at all.